RallySimulator.Domain/RallySimulator.Domain.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+	<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Comment:
Let's make the indent the same as it was before - I think it's 2 spaces

Reply:
Done